### ISSUE TITLE
ANDROID_CMAKE not defined.

### ISSUE THREAD

**Issue by kleysonr** [Luca96/dlib-for-android#1]
Hi,

I'm trying to use your setup.sh script, but the variable ANDROID_CMAKE is not defined.

I suppose that in the line https://github.com/Luca96/dlib-for-android/blob/master/setup.sh#L9 , we shoud have:

```
ANDROID_CMAKE='your-path-to/Android/sdk/cmake/3.10.2.4988404/bin/cmake'
```
instead of

```
AndroidCmake='your-path-to/Android/sdk/cmake/3.10.2.4988404/bin/cmake'
```

And lines https://github.com/Luca96/dlib-for-android/blob/master/setup.sh#L49 and https://github.com/Luca96/dlib-for-android/blob/master/setup.sh#L64 we should have

```$ANDROID_CMAKE ...```

instead of 

```ANDROID_CMAKE ...```

I had to do those changes to have the cmake working.

**Comment by Luca96** [comment]
Hi @kleysonr,
thanks for reporting the error. Sorry for the mistake, but I'm not able to test the bash script. 
I've make the changes you'd suggest in the lastest commit.
Moreover I find an error in a path. Maybe it can fix the issue #2 too.

Hope it works. 
Best Regards.

**Comment by kleysonr** [comment]
Fixed.
